src/main/java/athena/Storage.java

```diff
@@ -13,6 +13,8 @@
     private String filePath;
     private TaskList tasks;
     private Ui ui;
+
+
```

Comment:
1 empty line should be enough?

Reply:
will fix this

Reply:
oh this wasn't in my commit but was accidentally added onto this PR